c2corg_ui/static/js/user.js

```diff
@@ -113,6 +113,7 @@ app.UserController.prototype.errorLogout_ = function(response) {
   alert('logout error');
   console.log('logout error');
   console.log(response);
+  this.auth.removeUserData();
```

Comment:
So you remove the client-side user data even though the logout failed API side?
Does this mean that the logout will then be effective (since token is lost client side even though it is still available in the API DB)?